runtime/port/CMakeLists.txt

```diff
@@ -109,6 +146,16 @@ target_link_libraries(j9prt
 		j9thr
 		${CMAKE_DL_LIBS}
 )
+if(OMR_OS_WINDOWS)
+	target_link_libraries(j9prt
+		PRIVATE
+			Ws2_32.lib
+			shell32.lib
+			Iphlpapi.lib
+			psapi.lib
+			pdh.lib
```

Comment:
This list is larger than the list in the module.xml:
https://github.com/eclipse/openj9/blob/e2c4592f1b2ab0fcdca3d76e2414593863c58a0c/runtime/port/module.xml#L365-L376

Where does `Ws2_32.lib` come from?

Reply:
The socket library macro
https://github.com/eclipse/openj9/blob/e2c4592f1b2ab0fcdca3d76e2414593863c58a0c/runtime/port/module.xml#L355

The macro gets its definition from the `uma_socketLibraries` property in the build specs. Only windows specs  defines this property, and set it to `ws2_32.lib,Iphlpapi.lib`